### ISSUE TITLE
Stealth Detection

### ISSUE THREAD

**Issue by Jseay1313** [Atlantiss/NetherwingBugtracker#3993]
**Description**: Stealth detection is completely fucked on this server, mobs 1-3 levels higher than you popping you out of stealth 15-18 yards away, mobs that are 3-5 levels lower than you popping you out of stealth from 8 yards away.

**Current behaviour**: Stealth sucks, detection is way too high.

**Expected behaviour**: Stealth detection should function as normal, mobs the same or lower level than you should have a significantly reduced chance to detect you.

**Server Revision**: 

https://imgur.com/a/Dd36os7

**Comment by The14me** [comment]
need proof what is ''normal''

**Comment by Maikie1234** [comment]
Can confirm, stealth detection sucks sometimes, but i didnt find out how this occours. One time I am logged onto my rogue, stealth is eighter really good (I can literary stand in front of an enemy and i wont get detected) or it just straight up sucks (mobs/players can see me on miles away from them, even when im behind them). I tested this with a friend of mine, who plays on the opposite faction and is actually one level under me. One time i was standing right on his face and he wasnt able to see me. I tried reloging few times and repeating the process. Aprox. one time out of 4 tries, we managed to "get this bug to work". My friend could see me and target me when i was quite far away from him (aprox on ranged auto attacks range)
